### DISCUSSION TITLE
minpoll and iburst

### DISCUSSION THREAD

**Discussion by soa-jrt** [centerclick/feedback#116] [Q&A] [answered]
I'm making this a 'discussion' rather than an 'issue', because I'm not convinced there is an issue here.

With an NTP270 running 2.18.3, it appears to me that ntpd is using a minpoll and maxpoll set at "10" (1,204 seconds) for the 'backup servers'. I assume this by design, but don't understand why. Why not iburst with minpoll=6 and maxpoll=10 (or even 14 or 16)? ntpd is designed to increment the polling interval from short to long as the time signal stabilizes.

A shorter minpoll (and iburst) would let the signal-selections be made more quickly at startup. Especially useful in cases where satellites are invisible.
A longer maxpoll is entirely suitable for a stable NTP server. maxpoll=10 works out to about ~ 4 polls per hour. A stable clock could easily make do querying 3 or 4 different backups ~ 1 per hour (or less).

I have not seen any evidence that iburst is in use, but maybe I just not waited long enough for my polling interval to be incremented past 10 ?

**Comment by dave4445** [comment, marked as answer]
This is intentional, there is an interaction when using a PPS source where `ntpd` doesn't correctly auto-scale the backup servers polling interval which results in always using the minpoll value for them.  we use `minpoll=maxpoll=10`, but yes `iburst` is on:

`BACKUP_OPS="iburst prefer minpoll 10"`

**Reply by soa-jrt** [reply]
I figured there was a reasonable explanation for the min/max. Thank you.
I must have missed the iburst packets in my capture during testing. Wouldn't be the first time.
